cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ScalarFunctionsTest.scala

```diff
@@ -31,153 +31,177 @@ class ScalarFunctionsTest extends DocumentingTest {
+              #WHERE a.name = 'Alice'
+              #RETURN coalesce(a.hairColor, a.eyes)""".stripMargin('#'),
+      ResultAssertions((r) => {
           r.toList should equal(List(Map("coalesce(a.hairColor, a.eyes)" -> "brown")))
         })) {
         resultTable()
       }
     }
     section("endNode()", "functions-endnode") {
-      p(
-        """`endNode()` returns the end node of a relationship.""".stripMargin)
-      function("endNode(relationship)", "A Node.", ("relationship", "An expression that returns a relationship."))
+      p("The function `endNode()` returns the end node of a relationship.")
+      function("endNode(relationship)",
+        "A Node.",
+        ("relationship", "An expression that returns a relationship."))
       considerations("`endNode(null)` returns `null`.")
-      query(
-        """MATCH (x:Developer)-[r]-()
-          |RETURN endNode(r)""".stripMargin, ResultAssertions((r) => {
+      query("""MATCH (x:Developer)-[r]-()
+              #RETURN endNode(r)""".stripMargin('#'),
+      ResultAssertions((r) => {
           r.columnAs[Node]("endNode(r)").toList.map(_.getId) should equal(Array(2, 1))
         })) {
         resultTable()
       }
     }
     section("head()", "functions-head") {
-      p(
-        """`head()` returns the first element in a list.""".stripMargin)
-      function("head(list)", "The type of the value returned will be that of the first element of `list`.", ("list", "An expression that returns a list."))
-      considerations("`head(null)` returns `null`.", "If the first element in `list` is `null`, `head(list)` will return `null`.")
-      query(
-        """MATCH (a)
-          |WHERE a.name = 'Eskil'
-          |RETURN a.array, head(a.array)""".stripMargin, ResultAssertions((r) => {
-          r.columnAs[String]("head(a.array)").toList.head should equal("one")
+      p("The function `head()` returns the first element in a list.")
+      function("head(expression)",
+        "The type of the value returned will be that of the first element of the list.",
+        ("expression", "An expression that returns a list."))
+      considerations("`head(null)` returns `null`.",
+        "`head([])` returns `null`.",
+        "If the first element in `list` is `null`, `head(list)` will return `null`.")
+      query("""MATCH (a)
+              #WHERE a.name = 'Eskil'
+              #RETURN a.liked_colors, head(a.liked_colors)""".stripMargin('#'),
+      ResultAssertions((r) => {
+          r.columnAs[String]("head(a.liked_colors)").toList.head should equal("pink")
         })) {
         p("The first element in the list is returned.")
         resultTable()
       }
     }
     section("id()", "functions-id") {
-      p(
-        """`id()` returns the id of a relationship or node.""".stripMargin)
-      function("id(expression)", "An Integer.", ("expression", "An expression that returns a node or a relationship."))
+      p("The function `id()` returns the id of a relationship or node.")
```

Comment:
I have changed the text to:


The function `id()` returns a unique identifier, the function can be utilized for a relationship or a node.

Neo4j implements the id so that:
                         
Node:: Every node in a database has a unique identifier compared to other nodes in the same database, the uniqueness is guaranteed within the scope of a single transaction.

Relationship:: Every relationship in a database has a unique identifier compared to other relationships in the same database, the uniqueness is guaranteed within the scope of a single transaction.

Reply:
It is the same, no? You have changed id to identifier, or I am missing something. I have a problem with "compared to". It refers to every node. And, a unique ID is a unique ID, I don't think that further explanation is needed.